src/components/side_panel/pivot/pivot_title_section/pivot_title_section.ts

```diff
@@ -13,7 +13,7 @@ interface Props {
 
 export class PivotTitleSection extends Component<Props, SpreadsheetChildEnv> {
   static template = "o-spreadsheet-PivotTitleSection";
-  static components = { CogWheelMenu, Section, EditableName };
```

Comment:
Can delete the component `EditableName` entierely now, no ?

Reply:
It's still used for the lists in odoo. So it will be removed in my other task